apis/run/Makefile

```diff
@@ -0,0 +1,10 @@
+# Copyright 2022 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+include ../../common.mk
+# set root for manifests
+RELATIVE_ROOT=.
```

Comment:
Just wondering why we need to define ```RELATIVE_ROOT```?

Reply:
make target `generate-manifests` in common.mk is using it for output folder for crds